README.md

```diff
@@ -5,16 +5,23 @@ Application for demonstrating surface blending with ros
 
 Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C3%B6del
 
+### 3rd Party Dependencies
+==============
+- libqd-dev
+- [wstool](http://wiki.ros.org/wstool)
+
 
 ### Installation
 ==============
 
-Godel can be installed from source by cloning this repo.  Package specific dependencies/installation instructions can be found below.
-
- * [Polygon Offset Package](godel_polygon_offset/README.md)
- * [Process Path Generation](godel_process_path_generation/README.md) 
+Cd into the 'src' directory of your catkin workspace and run the following:
+```
+wstool init . https://github.com/ros-industrial-consortium/godel/raw/hydro-devel/godel.rosinstall
+cd ..
```

Comment:
We should add something like: `rosdep check --from-paths src --ignore-src` (or the `rosdep install ..` variant).

That would also automatically install `libqd-dev`, if users don't have it (see earlier comment).